acc_test/resource_cluster_test.go

```diff
@@ -231,6 +234,60 @@ func TestAccClusterCustomVPC(t *testing.T) {
 	})
 }
 
+func TestAccKafkaZookeeperContactPointsEven(t *testing.T) {
+	testAccProvider := instaclustr.Provider()
+	testAccProviders := map[string]terraform.ResourceProvider{
+		"instaclustr": testAccProvider,
+	}
+	validConfig, _ := ioutil.ReadFile("data/kafka_zk.tf")
+	username := os.Getenv("IC_USERNAME")
+	apiKey := os.Getenv("IC_API_KEY")
+	hostname := getOptionalEnv("IC_API_URL", instaclustr.DefaultApiHostname)
+	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey, hostname)
+	resource.Test(t, resource.TestCase{
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckResourceDeleted("kafka_zk", hostname, username, apiKey),
+		Steps: []resource.TestStep{
+			{
+				Config: oriConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckResourceValid("kafka_zk"),
+					testCheckResourceCreated("kafka_zk", hostname, username, apiKey),
+					testCheckContactIPCorrect("kafka_zk", hostname, username, apiKey, 3),
+				),
+			},
+		},
+	})
+
+}
+
+func TestAccCassandraSparkContactPointsOdd(t *testing.T) {
```

Comment:
these new tests seem to require more clusters that need to be in the RUNNING state. How much more time does that add to our acceptance test suite? we already had 4 that requires RUNNING clusters and they add significant amount of time to the testing

Reply:
They do add a fair bit. I could actually move the tests to the existing clusters that need to reach running?